tests/conftest.py

```diff
@@ -71,9 +73,21 @@ def set_backend(backend):
 
 
 def skip_np():
+    r"""
+    Skips tests if the backend is numpy.
+    """
     if math.backend_name == "numpy":
         pytest.skip("numpy")
 
 
+def skip_tf():
+    r"""
+    Skips tests if the backend is tensorflow and the version is ``>2.15.0``.
+    """
+    if math.backend_name == "tensorflow":
+        if Version(metadata.distribution("tensorflow").version) > Version("2.15.0"):
```

Comment:
Right now yes. I'm still debating on having 2.15.0 be the lowest but then you have to jump through some hoops to have `tensorflow-macos` install when `tf == 2.15.0` but not install if `tf>2.15.0`.

Reply:
hmm. we haven't released since March - @elib20 is there a plan to make another official MrMustard version? I'm wondering how bad it would be to just say "if using MrMustard 7.4+, then you need TF 2.16+"

Reply:
I think the next planned release is once lab dev is complete. Seeing as that would be a massive update from the previous release it's completely fine to mention in the CHANGELOG that we just bumped the version